code/ui/generalhud/nameplate/Nameplate.cs

```diff
@@ -85,18 +84,12 @@ public override void Tick()
         {
             base.Tick();
 
-            if (Local.Pawn is not TTTPlayer player)
-            {
-                return;
-            }
-
-            if (player.Camera is ThirdPersonCamera)
+            if (Local.Pawn is not TTTPlayer player || player.Camera is ThirdPersonCamera)
             {
                 return;
             }
 
             TraceResult trace = Trace.Ray(player.EyePos, player.EyePos + player.EyeRot.Forward * MAX_DRAW_DISTANCE)
-                .Ignore(player.ActiveChild)
```

Comment:
Instead, we should ignore the observed player if the current player is observing